addon/webextension/selector/ui.js

```diff
@@ -446,6 +448,7 @@ this.ui = (function() { // eslint-disable-line no-unused-vars
 
     unhide() {
       this.element.style.display = "";
+      catcher.watchPromise(callBackground("sendEvent", "internal", "unhide-preview-frame"));
```

Comment:
I'm a little concerned that callBackground is going to confound the timing measurements a little. But... eh?

Reply:
Yeah, the time to send a message seems pretty small, <=10ms (for short messages), and some of these steps take several seconds, so it seemed fine for now.